source/includes/steps-install-bi-connector.yaml

```diff
@@ -23,9 +23,8 @@ action:
 
   - heading: "Windows"
     pre: | 
-
-      Delete the MongoDB BI Connector directory from the :guilabel:`Program
-      Files` directory.
+      Delete the MongoDB BI Connector directory from the 
```

Comment:
If they used the MSI to install, then they should use it(the msi) or Add/Remove Programs to uninstall it.

Also, patch updates (2.2.1, 2.2.2, etc...) will install over the top, while major/minor updates will install side-by-side (2.3, 2.4, 3.0)